app/assets/javascripts/jquery.js

```diff
@@ -0,0 +1,4 @@
+import jquery from 'jquery';
```

Comment:
Why is this needed? Is this required by ActiveAdmin?
If the goal here is to be able to use jquery for our custom JS scripts, I'd like to not do that. I think we should start using Stimulus for those scenarios.

Reply:
It's used by activeadmin, same for jquery-ui. Some things don't work without it (like batch actions, datepicker, etc.)